draft-packages/tabs/KaizenDraft/Tabs/styles.scss

```diff
@@ -99,17 +106,67 @@
   }
 }
 
-.disabledTab {
+.disabledTabRow {
   composes: tab;
+  composes: row;
   cursor: inherit;
   opacity: 0.3;
+  font-weight: normal;
+
+  &.activeTabRow:hover {
+    background-color: transparent;
+  }
 
-  &:hover:not(.activeTab) {
+  &:hover:not(.activeTabRow) {
     color: $kz-color-wisteria-800;
     background-color: transparent;
+    font-weight: normal;
+  }
+}
+
+.row + .row {
+  @include ca-margin($start: $ca-grid * 0.5);
+}
+
+.column {
+  composes: tab;
+  min-height: $kz-spacing-xl;
+  display: flex;
+  align-items: center;
+  justify-content: left;
+  padding-left: $kz-spacing-sm;
+  border-radius: $kz-border-solid-border-radius;
+
+  :global(.js-focus-visible) &:global(.focus-visible):after {
+    border-radius: $kz-border-borderless-border-radius;
+    z-index: 1; // pop the border out so sibling tabs don't overlap
+  }
+
+  &.rtl {
```

Comment:
@sebpearce 's work on the the RTL stuff, but I believe the recommended a approach is to use a html attribute selector like `[dir="rtl"]` like in https://github.com/cultureamp/kaizen-design-system/blob/master/draft-packages/split-button/KaizenDraft/SplitButton/styles.scss#L151". Otherwise you'd have to prop drill the rtl prop down every component, which is impractical, and would have the likely outcome that no one would do it.

Reply:
Just noticed this in the GitHub Slack updates 😎  Yeah, the RTL prop is just for stuff that can't be done in CSS, like using a different icon. `dir="rtl"` works even if they don't use the prop 👌

Reply:
I'd like to eventually use `context` for stuff like this.

Reply:
I'm kinda in favour of the components having an optional `dir` prop, so they can override ltr/rtl behaviour for specific components if needed. I've kept the `dir` prop here, but this would also work if `dir` wasn't passed to the component and instead the layout set the direction.

Reply:
> but this would also work if dir wasn't passed to the component and instead the layout set the direction.

This is incorrect. I just tested it. Without touching any CSS: 
Layout => RTL = true
Tabs => RTL = false
![Screen Shot 2020-06-29 at 4 09 40 pm](https://user-images.githubusercontent.com/1621182/85978937-faf16580-ba22-11ea-8f49-289003861b38.png)

Layout => RTL = true
Tabs => RTL = true
This is what it _should_ look like for RTL languages
![Screen Shot 2020-06-29 at 4 09 29 pm](https://user-images.githubusercontent.com/1621182/85978942-fcbb2900-ba22-11ea-8c90-43b2eeef0a02.png)

Reply:
Having the base `[dir="rtl"]` selector as a safety net along with an RTL override class sounds good to me 👍

Reply:
BTW, in the past people have called that prop `textDirection` which is a bit more intuitive to people unfamiliar with the `dir` prop.

Reply:
Ah 😬 `Box` uses `rtl?: boolean`. I'm going to switch to `textDirection?: "ltr" | "rtl"` since that has way more usages.